toolkits/global/packages/global-javascript/src/helpers/util/onetrust.js

```diff
@@ -0,0 +1,36 @@
+import {getCookie} from './get-cookie';
+
+/**
+ * onetrust.js
+ * @param {string} category - name of OneTrust category
+ * @return {boolean}
+ */
+
+const checkConsent = category => {
+	const validCategories = {
+		strictlyNecessary: 'C0001',
+		performance: 'C0002',
+		functional: 'C0003',
+		targetingFirstParty: 'C0008',
+		targetingThirdParty: 'C0009'
+	};
+
+	if (!validCategories[category]) {
+		throw new Error('Invalid category: ' + category);
+	}
+
+	const consent = getCookie('OptanonConsent');
+	const consentGroups = consent.split('groups=').pop().split('&')[0];
+	const colonEncoding = '%3A';
+
```

Comment:
Is it likely that any of this could change? either by us or onetrust. i.e. would it be valuable to make some of this configurable with an `init()` function, such as categories, cookie name? It might be that we hope not, and just cross that bridge when/if it happens, but it's easier for an app to update a config than release a new version of this package every time something changes.

Reply:
Hmm good question. I think that it's unlikely that the cookie name would change, but it's possible that the categories could change (based on our side). I could make the categories I've set here the default option and allow different categories to be set via an init.

Reply:
If that's easy then it would safeguard against them changing

Reply:
After discussion offline, we think it is best to leave it as it is. We want the module to reflect the current state of our OneTrust configuration, and making it configurable would mean those categories _could_ go stale. Every team should be using the same set of categories; if they change for one they change for all. So a new version will be released if the categories change, and teams will update.